packages/client/src/objectSet/ObjectSetListenerWebsocket.ts

```diff
@@ -454,7 +454,12 @@ export class ObjectSetListenerWebsocket {
 
     for (const osdkObject of osdkObjectsWithReferenceUpdates) {
       if (osdkObject != null) {
-        sub.listener.onChange?.(osdkObject);
+        try {
+          sub.listener.onChange?.(osdkObject);
+        } catch (error) {
+          this.#logger?.error(error, "Error in onChange callback");
+          sub.listener.onError({ subscriptionClosed: false, error });
```

Comment:
if an error happens while in `onError` the whole thing still breaks, no? We need a fallback in that case.